docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -2,111 +2,151 @@
+consensus failures, reward losses, and general instability.
+::::
 
-- **Both a validator node and a validator fullnode required:** For the Aptos mainnet, we require that you run a validator node and a validator fullnode. We strongly recommend that you run the validator node and the validator fullnode on two separate and independent machines. Make sure that these machines are well-provisioned and isolated from each other. Guaranteeing the resource isolation between the validator and the validator fullnode will help ensure smooth deployment of these nodes.
-- **Public fullnode is optional:** We recommend that optionally you run a public fullnode also. However, a public fullnode is not required. If you run public fullnode also, then we strongly recommend that you run the public fullnode on a third machine that is separate and independent of either the validator or the validator fullnode machines.
-  :::tip Terraform support
-  For deploying the nodes in cloud we have provided Terraform support on two cloud providers: **GCP** and **AWS**. See [**Running Validator Node**](running-validator-node/index.md).
-  :::
+## Resource isolation
 
-- **Open the network ports:** Make sure that you open the network ports prior to connecting to the network. See [Ports](#ports).
-- **Close the network ports:** Make sure that you close these ports after either being accepted or rejected for the network.
+When running an Aptos validator and VFN, we strongly recommend that the nodes run on two separate and
+independent machines. These machines should be well-provisioned, meet the requirements outlined below and
+be isolated from each other. Maintaining resource isolation between the validator and the VFN is important
+for security and to ensure that the nodes do not encounter performance degradation, instability or failures when
+under load.
+
+:::tip Terraform support
+For deploying the validator and VFN in the cloud, we provide Terraform support on two cloud providers: **GCP** and **AWS**. See [**Running Validator Node**](running-validator-node/index.md).
+:::
 
 ## Hardware requirements
 
-For running an Aptos **validator node and validator fullnode** we recommend the following hardware resources:
-For running a validator and validator fullnode is that your hardware spec should be good enough to achieve a 30k peak
-TPS on mainnet.
+For running an Aptos validator and VFN in mainnet, we recommend that your hardware be performant enough to maintain
+~30,000 transactions per second (TPS). There are two ways to evaluate if your hardware meets this requirement:
 
-In order to evaluate that, checkout aptos-core repo and install required dependencies (See [**cloning aptos-core**](https://aptos.dev/guides/building-from-source#clone-the-aptos-core-repo)).
-Use following commands to run the performance benchmark
+1. Use the reference specs provided below.
```

Comment:
There should be a note regarding the possible variance in compute complexity of transactions (by orders of magnitude), to let the partners know why the specs below are not overkill (since it's been often asked on Slack and Discord why the specs are so high since most of the time the usage is <10%).

Reply:
Done -- added a section to the hardware motivation explaining why folks need to provision for the worst case 😄